docs/tutorial/sql.md

```diff
@@ -656,6 +656,37 @@ Since v`1.3.0`, Sedona natively supports writing GeoParquet file. GeoParquet can
 df.write.format("geoparquet").save(geoparquetoutputlocation + "/GeoParquet_File_Name.parquet")
 ```
 
+Since v`1.5.1`, Sedona supports writing GeoParquet files with custom GeoParquet spec version and crs.
+The default GeoParquet spec version is `1.0.0` and the default crs is `null`. You can specify the GeoParquet spec version and crs as follows:
+
+```scala
+val projjson = "{...}" // PROJJSON string for all geometry columns
+df.write.format("geoparquet")
+		.option("geoparquet.version", "1.0.0")
+		.option("geoparquet.crs", projjson)
```

Comment:
Can you also add some text explaining the behavior of Sedona.

"geoparquet.crs","" will generate a geoparquet file without the `crs` field meaning this is CRS84
"geoparquet.crs","null" will generate a geoparquet file with `null` in its `crs` field meaning the CRS is unknown